OWNERS_ALIASES

```diff
@@ -11,34 +11,25 @@ aliases:
   - strongjz
 
   ingress-nginx-maintainers:
-  - rikatz
-  - strongjz
```

Comment:
this should keep

Reply:
Aren't they covered by `ingress-nginx-admins`? I also added that alias to global `OWNERS`, so it should result in the same, right?

Reply:
Sorted stuff in recent update, so gonna resolve that one.